rootfs/scheduler/k8s.py

```diff
@@ -1,18 +1,22 @@
-import copy
-import httplib
 import json
+import logging
 import random
 import re
 import string
 import time
+import urlparse
 
 from django.conf import settings
 from docker import Client
 from .states import JobState
 from . import AbstractSchedulerClient
+import requests
 
+logger = logging.getLogger(__name__)
 
-POD_TEMPLATE = '''{
+
+POD_TEMPLATE = """\
+{
   "kind": "Pod",
   "apiVersion": "$version",
```

Comment:
Not that this needs to be done in this commit... but there is no real point in adding $version as a variable, as the version string indicates the version of the file format. The only case in which the version would change is if you changed the formatting for the POD_TEMPLATE variable. (I think this is actually left-over from the k8s preview release.)